### PR TITLE
housekeeping: Improve docs and make help

### PR DESCRIPTION
**Pull request by juliaogris**
- Document release process.
- Make first README.md link point to evy.dev rather than scratch.
- Replace sponsorship red heart emoji with a white one for readability.

Update `make help` awk scripts for multi-line comments on top or rather than
next to target.

Add `/version` endpoint to deployment.

---

Fixes evylang/todo#2
Fixes evylang/todo#3
Fixes evylang/todo#5
Fixes evylang/todo#7
Fixes evylang/todo#8

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -28,23 +28,26 @@ CMDS = .
...
+install:
 	go install -ldflags='$(GO_LDFLAGS)' $(CMDS)
 
 # Use `go version` to ensure the right go version is installed when using tinygo.
 go-version:
 	go version
 
-# Optimise tinygo output for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
-tiny: go-version | $(O) ## Build for tinygo / wasm
+## Build for tinygo / wasm for size, see https://www.fermyon.com/blog/optimizing-tinygo-wasm
```
Comment (camh-) [inline Makefile:39]: Should that `see https://...` stuff really be part of the help message? If not, you can put it on the next line with a single `#` comment marker. or you can put it on the line before - again, single comment marker.

File: docs/development/releasing.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+To trigger a **minor** version bump a new file must be added to the
+[release-notes directory]. A minor version bump signifies the completion of a
+milestone. The release notes file is named according to the new version, for
+example `v0.2.0.md`.
+
+Evy's current major version is still 0, this may change in the future.
+Evy's language syntax, tooling, and public API are still under development
+and not yet considered stable. Additionally, for major version 0, the meaning
+of patch and minor versions does not conform to the standard semantic
```
Comment (camh-) [inline docs/development/releasing.md:24]: perhaps link to https://semver.org/ for "semantic versioning specification"?

**Review by camh- — APPROVED** [review]
LGTM :green_apple: with a couple of minor comments

File: Makefile (hunk 1 above)
Comment (juliaogris) [reply Makefile]: done.

File: docs/development/releasing.md (hunk 2 above)
Comment (juliaogris) [reply docs/development/releasing.md]: done.

**Review by camh- — APPROVED** [review]
:green_apple:

**Pull request merged**